components/ipfs/ipfs_utils_unittest.cc

```diff
@@ -700,3 +700,15 @@ TEST_F(IpfsUtilsUnitTest, IsValidCIDOrDomain) {
   ASSERT_FALSE(ipfs::IsValidCIDOrDomain("a.b.c.com:11112"));
   ASSERT_FALSE(ipfs::IsValidCIDOrDomain("wrongdomainandcid"));
 }
+
+TEST_F(IpfsUtilsUnitTest, ToConfiguredGatewayURL) {
```

Comment:
need a test case which fails because ParseCIDAndPathFromIPFSUrl fails.

Reply:
To satisfy this I think I need a URL that (1) has an ipfs scheme, and (2) has a host.  So far, I've been unable to a test GURL that has these properties after being parsed.